packages/serve/src/lib/server.ts

```diff
@@ -1,24 +1,34 @@
+import { isEmpty } from 'lodash';
+import * as fs from 'fs';
+import * as http from 'http';
+import * as https from 'https';
 import {
   VulcanArtifactBuilder,
   TYPES as CORE_TYPES,
   CodeLoader,
 } from '@vulcan-sql/core';
-import * as http from 'http';
 import { Container, TYPES } from '../containers';
 import { ServeConfig } from '../models';
 import { VulcanApplication } from './app';
+import { getEnforceHttpsOptions } from './middleware';
 export class VulcanServer {
   private config: ServeConfig;
   private container: Container;
-  private server?: http.Server;
-
+  private servers?: {
+    http: http.Server;
+    https?: https.Server;
+  };
   constructor(config: ServeConfig) {
     this.config = config;
     this.container = new Container();
   }
-
-  public async start(port = 3000) {
-    if (this.server)
+  /**
+   * Start the vulcan server
+   * @param port the http port for server start, default is 3000
+   * @param httpsPort the https port for https server start when you set "type" = LOCAL in "enforce-https" middleware and provide ssl file, default port is 3001
+   */
+  public async start(port = 3000, httpsPort = 3001) {
```

Comment:
Thanks @oscar60310 for reviewing and suggestion, I have moved the `httpsPort` setting into config under `enforce-https` options, because the https server is only created when `enforce-https` set `LOCAL` type.

Btw, after I moved to the config, seems CLI no need to update too

Reply:
Hi @oscar60310, thanks for discussing with me, I have also move the http port into config for reading!